sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java

```diff
@@ -135,16 +135,36 @@
 import org.junit.rules.TestRule;
 import org.junit.runner.Description;
 import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
+import org.junit.runners.Parameterized;
```

Comment:
I didn't look into tests in detail yet but given the extremely large change and number of potential users, we should try to get close to 100% test coverage and 100% test parity with streaming inserts (both unit and integration).

Reply:
AFIACT all tests are run against the new path, except for the ones that don't make sense (e.g. the ones that test specific functionality that doesn't exist in the storage api path)